docs/animations/AnimationSet.md

```diff
@@ -43,8 +43,6 @@ anim.Start();
 
 ## Properties
 
-
-
```

Comment:
General improvement. So you can revert it.

Reply:
Will you apply this fix to all docs it affects?

Reply:
Yes